iphone/Classes/TiUIiOSStepper.m

```diff
@@ -0,0 +1,204 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2009-2016 by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+#ifdef USE_TI_UIIOSSTEPPER
+#import "TiUIiOSStepper.h"
+#import "TiUIiOSStepperProxy.h"
+#import "TiUtils.h"
+
+@implementation TiUIiOSStepper
+
+-(UIStepper*)stepper
+{
+    if (stepper ==nil)
+    {
+        stepper = [[[UIStepper alloc]init] retain];
+        [stepper setMaximumValue:100];
+        [stepper setMinimumValue:0];
+        [stepper addTarget:self action:@selector(stepperChanged:) forControlEvents:UIControlEventValueChanged];
+        [stepper addTarget:self action:@selector(stepperTouch:forEvent:) forControlEvents:UIControlEventTouchDown];
+        [self addSubview:stepper];
+    }
+    return stepper;
+}
+
+-(void)dealloc
+{
+    [stepper removeTarget:self action:@selector(stepperChanged:) forControlEvents:UIControlEventValueChanged];
+    [stepper removeTarget:self action:@selector(stepperTouch:forEvent:) forControlEvents:UIControlEventTouchDown];
+    RELEASE_TO_NIL(stepper);
+    RELEASE_TO_NIL(backgroundImageCache);
+    [super dealloc];
+}
+-(void)setContinuous_:(id)value
+{
+    [[self stepper] setContinuous:[TiUtils boolValue:value]];
+}
+
+-(void)setAutorepeat_:(id)value
+{
+    [[self stepper] setAutorepeat:[TiUtils boolValue:value]];
+}
+
+-(void)setWraps_:(id)value
+{
+   [[self stepper] setWraps:[TiUtils boolValue:value]];
+}
+
+-(void)setMaximumValue_:(id)value
+{
+    ENSURE_TYPE(value, NSNumber)
+    [[self stepper] setMaximumValue:[TiUtils doubleValue:value]];
+}
+
+-(void)setMinimumValue_:(id)value
+{
+    ENSURE_TYPE(value, NSNumber)
+    [[self stepper] setMinimumValue:[TiUtils doubleValue:value]];
+}
+
+-(void)setStepValue_:(id)value
+{
+    ENSURE_TYPE(value, NSNumber)
+    NSNumber *stepValue = (NSNumber*)value;
```

Comment:
Pretty sure you can, since it calls the double-initializer. We do that in many places.